python/feathub/tests/mysql_test_base.py

```diff
@@ -0,0 +1,47 @@
+#  Copyright 2022 The Feathub Authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+import unittest
+from typing import Optional
+
+from testcontainers.mysql import MySqlContainer
+
+_mysql_container: Optional[MySqlContainer] = None
+
+
+class MySQLITTestBase(unittest.TestCase):
+    """
+    Base class for MySQL IT test. It reuses a global mysql container for all the MySQL
+    IT test cases.
+    """
+
+    mysql_container: Optional[MySqlContainer] = None
```

Comment:
Would it be simpler to do the following so that we don't a dedicated class for each container (e.g. redis, mysql)? 

- Add the static variable `FeathubITTestBase#mysql_container`.
- Add static method `FeathubITTestBase#setup_mysql_container` and `FeathubITTestBase#teardown_mysql_container`
- Let `MySQLSourceSinkITTest#setUpClass` invoke `setup_mysql_container`. And let `MySQLSourceSinkITTest#teardownUpClass` invoke `teardown_mysql_container`.